cypress.config.ts

```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from "cypress";
 import { createRequire } from "module";
+import getCompareSnapshotsPlugin from 'cypress-image-diff-js/plugin';
```

Comment:
Is there a simpler solution without image diff? Checking that there's no error in case the file type is OK and check that the error appears when the file isn't?